### PR TITLE
[enh] add example to use '-a' option with 'yunohost app install' comm…

### PR DESCRIPTION
**Pull request by M5oul**
…and.

I can't use a dot to split in two sentences.

### CODE REVIEW

File: data/actionsmap/yunohost.yml [hunk 1]
```diff
@@ -436,7 +436,7 @@ app:
                     help: Custom name for the app
                 -a:
                     full: --args
-                    help: Serialize arguments for app installation
+                    help: Serialized arguments for app script (i.e. "domain=domain.tld&url_path=/path")
```
Comment (opi) [inline data/actionsmap/yunohost.yml]: why use "url_path" instead of the commonly used "path" option ? ;)

**Comment by opi** [comment]
thx

File: data/actionsmap/yunohost.yml (hunk 1 above)
Comment (M5oul) [inline data/actionsmap/yunohost.yml]: I prefer use [`url_path`](https://github.com/M5oul/sonerezh_ynh/blob/master/scripts/install#L6) and [`src_path`](https://github.com/M5oul/sonerezh_ynh/blob/master/scripts/install#L34) variables to be more specific and avoid confusions between `path` and `final_path` variables in the install script.

Comment (opi) [inline data/actionsmap/yunohost.yml]: But "domain" and "path" are kind of reserved variable names, and are automatically saved on app_install ( https://github.com/yunohost/yunohost/blob/unstable/src%2Fyunohost%2Fapp.py#L876 ).
Using `url_path` could lead to some variable name mixin as in https://github.com/M5oul/sonerezh_ynh/blob/master/scripts%2Fupgrade#L9
But your app, your choice ;)

Comment (M5oul) [inline data/actionsmap/yunohost.yml]: Yep, you are right, we should call this variable `path`.
Fixed with https://github.com/YunoHost/yunohost/commit/1dd1ae483f1308a836af3cf4a8c9403ba26a591e.

Comment (opi) [inline data/actionsmap/yunohost.yml]: thx for that. Better cli help description is always helpfull

**Pull request merged**